app/workers/process_photo.rb

```diff
@@ -15,6 +15,15 @@ def perform(id)
 
       photo.processed_image.store!(unprocessed_image)
 
+      ##unless current_user.strip_exif
```

Comment:
- Incorrect indentation detected (column 6 instead of 4).
- Missing space after #.